sktime/forecasting/base/_base.py

```diff
@@ -448,6 +449,111 @@ def predict_interval(
 
         return pred_int
 
+    def predict_var(
+        self,
+        fh=None,
+        X=None,
+    ):
+        """Compute/return variance forecasts.
+
+        State required:
+            Requires state to be "fitted".
+
+        Accesses in self:
+            Fitted model attributes ending in "_".
+            self.cutoff, self._is_fitted
+
+        Writes to self:
+            Stores fh to self.fh if fh is passed and has not been passed previously.
+
+        Parameters
+        ----------
+        fh : int, list, np.array or ForecastingHorizon
+            Forecasting horizon, default = y.index (in-sample forecast)
+        X : pd.DataFrame, optional (default=None)
+            Exogenous time series
+
+        Returns
+        -------
+        pred_var : pd.DataFrame
+            Column names are exactly those of `y` passed in `fit`/`update`.
+                For nameless formats, column index will be a RangeIndex.
+            Row index is fh. Entries are variance forecasts, for var in col index.
+        """
+        if not self.get_tag("capability:pred_int"):
+            raise NotImplementedError(
+                f"{self.__class__.__name__} does not have the capability to return "
+                "variance predictions. If you "
+                "think this estimator should have the capability, please open "
+                "an issue on sktime."
+            )
+        self.check_is_fitted()
+        # input checks
+        fh = self._check_fh(fh)
+
+        # check and convert X
+        X_inner = self._check_X(X=X)
+
+        pred_var = self._predict_var(fh=fh, X=X_inner)
+
+        return pred_var
```

Comment:
maybe stupid question, but how many columns does this `pd.DataFrame` have? Can it also be a ´pd.Series´?

Reply:
it could, but I thought we were moving to "one output type", so it´s currently always a `pd.DataFrame`.

Reply:
but we still use `pd.Series` for univariate, so I would expect also a `pd.Series` here if I give a `pd.Series`?

Reply:
now I am a bit confused, I though we use "what goes in comes out"? So why do we fix here to `pd.DataFrame`? Should it then not also support `numpy` and `pd.Series`?

Reply:
>now I am a bit confused, I though we use "what goes in comes out"?

We decided we have a target of "one output type", while the status quo is mostly "what goes in comes out".
While existing interfaces may take time to refactor, change, or deprecate, we can easily ensure that new interfaces satisfy the design principle.

Reply:
can we not use the "dtype machinery" also for `predict_var`? Then we can deprecate all together at some point in time?

Reply:
yes, we could, and it would be pretty easy...
although the "sister functions" `predict_interval` and `predict_quantiles` return only `pd.DataFrame`, since they need to use a `pd.MultiIndex` to structure the data.
So not only would it be a step back, it might also be confusing due to the inconsistency with the other "probabilistic prediction" functions.